### PR TITLE
make management command processes standalone

### PR DESCRIPTION
**Pull request by snopoke**
##### SUMMARY
This combines the 'celery' and 'pillowtop' management command processes into a generic management commands processes task. Deployment of these management commands are now fully controlled via the 'app-processes.management_commands` config.

##### ENVIRONMENTS AFFECTED
ALL

##### ISSUE TYPE
- Change Pull Request

##### COMPONENT NAME
Management commands

##### ADDITIONAL INFORMATION
Move the following management command processes into `app-processes.management_commands`:
* run_form_websocket_feed
* run_pillow_retry_queue

### CODE REVIEW

File: src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2 [hunk 1]
```diff
@@ -291,7 +291,7 @@ USER_REPORTING_METADATA_UPDATE_FREQUENCY = {{ localsettings.USER_REPORTING_METAD
 SMS_QUEUE_ENABLED = {{localsettings.get('SMS_QUEUE_ENABLED', False)}}
 REMINDERS_QUEUE_ENABLED = {{localsettings.get('REMINDERS_QUEUE_ENABLED', False)}}
 
-PILLOW_RETRY_QUEUE_ENABLED = {{ localsettings.get('PILLOW_RETRY_QUEUE_ENABLED', False) }}
+PILLOW_RETRY_QUEUE_ENABLED = True
```
Comment (emord) [inline src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2]: Will there be a followup to remove this after https://github.com/dimagi/commcare-hq/pull/25126/ ?

Comment (snopoke) [reply src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2]: yes

File: environments/echis/app-processes.yml [hunk 2]
```diff
@@ -9,6 +9,8 @@ management_commands:
     queue_schedule_instances:
     handle_survey_actions:
     run_sms_queue:
+    run_form_websocket_feed:
```
Comment (snopoke) [inline environments/echis/app-processes.yml]: @emord I'm also thinking of killing this - as far as I can tell it's only used by dimagisphere

Comment (snopoke) [reply environments/echis/app-processes.yml]: cfd4591

**Pull request merged**